### PR TITLE
Fix up all comments to be in the proper order with proper spacing.

### PR DESCRIPTION
**Pull request by JamesWTruher**
# PR Summary

This will fix issues we have with our autogenerated documentation.

## PR Context

Some documentation created by automatic generation has issues because:

- some of tags are not in the appropriate location
- some tags have spelling errors
- some tags do not have consistent spacing

This PR addresses those issues

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [ ] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the title (the `WIP` bot will keep its status check at `Pending` while the prefix is present) and remove the prefix when the PR is ready.
- **[Breaking changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)**
    - [x] None
    - **OR**
    - [ ] [Experimental feature(s) needed](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/main/reference/7.3/Microsoft.PowerShell.Core/About/about_Experimental_Features.md)
        - [ ] Experimental feature name(s): 
- **User-facing changes**
    - [x] Not Applicable
    - **OR**
    - [ ] [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed: 
- **Testing - New and feature**
    - [ ] N/A or can only be tested interactively
    - **OR**
    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
- **Tooling**
    - [x] I have considered the user experience from a tooling perspective and don't believe tooling will be impacted.
    - **OR**
    - [ ] I have considered the user experience from a tooling perspective and opened an issue in the relevant tool repository. This may include:
        - [ ] Impact on [PowerShell Editor Services](https://github.com/PowerShell/PowerShellEditorServices) which is used in the [PowerShell extension](https://github.com/PowerShell/vscode-powershell) for VSCode
        (which runs in a different PS Host).
            - [ ] Issue filed: 
        - [ ] Impact on Completions (both in the console and in editors) - one of PowerShell's most powerful features.
            - [ ] Issue filed: 
        - [ ] Impact on [PSScriptAnalyzer](https://github.com/PowerShell/PSScriptAnalyzer) (which provides linting & formatting in the editor extensions).
            - [ ] Issue filed: 
        - [ ] Impact on [EditorSyntax](https://github.com/PowerShell/EditorSyntax) (which provides syntax highlighting with in VSCode, GitHub, and many other editors).
            - [ ] Issue filed:

### CODE REVIEW

**Review by sdwheeler — APPROVED** [review]
This looks good to me. I see that you found many more issues in the tags (typos, missing tags, etc.). This cleanup greatly improves the docs.

File: src/Microsoft.Management.Infrastructure.CimCmdlets/CimSessionProxy.cs [hunk 1]
```diff
@@ -2154,12 +2154,11 @@ internal class CimSessionProxyNewCimInstance : CimSessionProxy
 
         /// <summary>
         /// Initializes a new instance of the <see cref="CimSessionProxyNewCimInstance"/> class.
-        /// <summary>
+        /// </summary>
         /// <remarks>
         /// Create <see cref="CimSession"/> by given computer name.
         /// Then create wrapper object.
         /// </remarks>
-        /// <param name="computerName"></param>
```
Comment (SteveL-MSFT) [inline src/Microsoft.Management.Infrastructure.CimCmdlets/CimSessionProxy.cs:2162]: This method has two params that aren't doc'd?

File: src/Microsoft.Management.Infrastructure.CimCmdlets/CimSessionProxy.cs [hunk 2]
```diff
@@ -2169,6 +2168,7 @@ public CimSessionProxyNewCimInstance(string computerName, CimNewCimInstance oper
         /// <summary>
         /// Initializes a new instance of the <see cref="CimSessionProxyNewCimInstance"/> class.
         /// </summary>
+        /// <param name="computerName"></param>
```
Comment (SteveL-MSFT) [inline src/Microsoft.Management.Infrastructure.CimCmdlets/CimSessionProxy.cs:2171]: This method doesn't have a `computerName` param, that's the one above and the second param is missing?

**Review by SteveL-MSFT — CHANGES_REQUESTED** [review]
If the intent of this PR is formatting changes including moving params to the right place, will there be a subsequent PR to fix the params as many are wrong?   I do think that should be a separate PR, but because this PR has some param changes, that's not clear of the intent.

**Review by SteveL-MSFT — APPROVED** [review]
We discussed this and this will be a 3 stage approach:
- fix syntactical problems with the comment based doc XML (this PR)
- fix the proper <param> elements (follow-up PR)
- fix the content (separate effort)

**Comment by iSazonov** [comment]
To exclude regressions we need to enable appropriate analyzer rules for doc comments.

File: src/System.Management.Automation/namespaces/TransactedRegistryKey.cs [hunk 3]
```diff
@@ -341,11 +341,11 @@ public void Dispose()
         /// <summary>
         /// <para>Creates a new subkey, or opens an existing one.
         /// Utilizes Transaction.Current for its transaction.</para>
+        /// </summary>
         /// <param name='subkey'>Name or path to subkey to create or open. Cannot be null or an empty string,
-        /// otherwise an ArgumentException is thrown.</param>
         /// <returns>A TransactedRegistryKey object for the subkey, which is associated with Transaction.Current.
         /// returns null if the operation failed.</returns>
-        /// </summary>
+        /// otherwise an ArgumentException is thrown.</param>
```
Comment (iSazonov) [inline src/System.Management.Automation/namespaces/TransactedRegistryKey.cs:348]: Nested `returns` above.

File: src/System.Management.Automation/namespaces/TransactedRegistryKey.cs [hunk 4]
```diff
@@ -897,9 +897,9 @@ internal TransactedRegistryKey InternalOpenSubKey(string name, bool writable)
         /// <summary>
         /// <para>Retrieves a subkey for readonly access.
         /// Utilizes Transaction.Current for its transaction.</para>
-        /// <param name="name">Name or path of the subkey to open.</param>
-        /// <returns>The subkey requested or null if the operation failed.</returns>
         /// </summary>
+        /// <returns>The subkey requested or null if the operation failed.</returns>
+        /// <param name="name">Name or path of the subkey to open.</param>
```
Comment (iSazonov) [inline src/System.Management.Automation/namespaces/TransactedRegistryKey.cs:902]: Better swap the lines.

File: src/System.Management.Automation/namespaces/TransactedRegistryKey.cs [hunk 5]
```diff
@@ -1107,9 +1107,9 @@ public string[] GetValueNames()
         /// doesn't exist. Utilizes Transaction.Current for its transaction.</para>
         /// <para>Note that name can be null or "", at which point the
         /// unnamed or default value of this Registry key is returned, if any.</para>
-        /// <param name="name">Name of value to retrieve.</param>
-        /// <returns>The data associated with the value.</returns>
         /// </summary>
+        /// <returns>The data associated with the value.</returns>
+        /// <param name="name">Name of value to retrieve.</param>
```
Comment (iSazonov) [inline src/System.Management.Automation/namespaces/TransactedRegistryKey.cs:1112]: Better swap the lines.

File: src/System.Management.Automation/namespaces/TransactedRegistryKey.cs [hunk 6]
```diff
@@ -1121,10 +1121,10 @@ public object GetValue(string name)
         /// doesn't exist. Utilizes Transaction.Current for its transaction.</para>
         /// <para>Note that name can be null or "", at which point the
         /// unnamed or default value of this Registry key is returned, if any.</para>
+        /// </summary>
+        /// <returns>The data associated with the value.</returns>
```
Comment (iSazonov) [inline src/System.Management.Automation/namespaces/TransactedRegistryKey.cs:1125]: The same - `returns` is usually last doc comment.

File: src/System.Management.Automation/namespaces/TransactedRegistryKey.cs [hunk 7]
```diff
@@ -1136,12 +1136,12 @@ public object GetValue(string name, object defaultValue)
...
```
Comment (iSazonov) [inline src/System.Management.Automation/namespaces/TransactedRegistryKey.cs:1140]: The same.

File: src/System.Management.Automation/namespaces/TransactedRegistryKey.cs [hunk 8]
```diff
@@ -1307,9 +1307,9 @@ internal object InternalGetValue(string name, object defaultValue, bool doNotExp
         /// <summary>
         /// <para>Retrieves the registry data type of the value associated with the specified name.
         /// Utilizes Transaction.Current for its transaction.</para>
-        /// <param name="name">The value name whose data type is to be retrieved.</param>
-        /// <returns>A RegistryValueKind value representing the registry data type of the value associated with name.</returns>
         /// </summary>
+        /// <returns>A RegistryValueKind value representing the registry data type of the value associated with name.</returns>
```
Comment (iSazonov) [inline src/System.Management.Automation/namespaces/TransactedRegistryKey.cs:1311]: The same.

File: src/System.Management.Automation/namespaces/TransactedRegistrySecurity.cs [hunk 9]
```diff
@@ -248,15 +248,15 @@ public override AccessRule AccessRuleFactory(IdentityReference identityReference
         /// <summary>
         /// <para>Creates a new audit rule, specifying the user the rule applies to, the access rights to audit, the inheritance and propagation of the
         /// rule, and the outcome that triggers the rule.</para>
+        /// </summary>
+        /// <returns>A TransactedRegistryAuditRule object representing the specified audit rule for the specified user, with the specified flags.
```
Comment (iSazonov) [inline src/System.Management.Automation/namespaces/TransactedRegistrySecurity.cs:252]: The same.

Comment (JamesWTruher) [reply src/System.Management.Automation/namespaces/TransactedRegistrySecurity.cs:252]: We're going to take a concerted approach to reorder all of the triple-slash comments as a separate approach. I've opened #18744 to track

File: src/System.Management.Automation/namespaces/TransactedRegistryKey.cs (hunk 3 above)
Comment (JamesWTruher) [reply src/System.Management.Automation/namespaces/TransactedRegistryKey.cs]: fixed

**Pull request merged**